### DISCUSSION TITLE
Error:Queue is full for system log

### DISCUSSION THREAD

**Discussion by shelly-ctrl** [ClickHouse/ClickHouse#58644] [Q&A]
Question:
clickhouse-version: 22.5.3
Run clickhouse-client --password ，after enter password ，unable to access database 

clickhouse hung while running and the clickhouse-server.err.log reported the following error:
2024.01.10 09:35:23.012096 [ 933940 ] {} <Error> SystemLog (system.asynchronous_metric_log): Queue is full for system log 'DB::AsynchronousMetricLog' at 1520599569

At present, the solution can only be restarted，However, after running for a period of time, this error will still appear, how to completely solve this problem?

Thanks!!!

**Comment by tcpayne** [comment]
Hi @shelly-ctrl - I am seeing this log error with hung sessions also.  So I am wondering what was the resolution for this ticket?  Thanks!

**Reply by shelly-ctrl** [reply]
upgrade clickhouse-server

**Comment by foxixianyu777** [comment]
which clickhouse-version do you upgrade? I have the same question.
